scripts/ansible/roles/linux/docker/vars/ubuntu/focal.yml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+---
+ci_apt_packages:
+  - "automake"
+  - "dh-exec"
+  - "dpkg-dev"
+  - "gawk"
+  - "git"
+  - "libmbedtls10"
```

Comment:
It seems this package is not available via default repos for focal. However, libmbedtls12 is